common/src/main/java/com/wynntils/services/mapdata/providers/builtin/WaypointsProvider.java

```diff
@@ -12,6 +13,7 @@
 
 public class WaypointsProvider extends BuiltInProvider {
     private static final List<MapFeature> PROVIDED_FEATURES = new ArrayList<>();
+    private static final List<MapIcon> PROVIDED_ICONS = new ArrayList<>();
```

Comment:
What we can do is however, provide these icons in a separate provider. Perhaps, the "personal settings provider" we don't have yet. We'll come back to it.

Reply:
In general, I think this is a good idea. That way, people can like download "icon packs" that provide additional icons, and if they want to add new ones of their own, they will essentially be starting to produce their own icon pack.

In fact, when thinking of it, maybe we should *only* support this method of setting icons for waypoints in the GUI.